### PR TITLE
Merge 86-correct-footer-horizontal-margin into develop

### PR DESCRIPTION
**Pull request by JacobHeldt**
## What

The padding in the footer was adjusted to align with the padding in other sections of the homepage (150px). Additionally, the spacing of items in the footer was adjusted, which results in a slightly altered design on desktop devices. The social media icons were also adjusted to be coherent with our marketing strategy.

## Why

The padding on the footer did not align with the padding of other sections. Furthermore, team members expressed concern about the design of the footer section on desktop devices.

## How

Edited Footer.css; Replaced certain social media icons with other social media icons.

## Related Issues

This pull request is focused on desktop and does not take into account the mobile design. The mobile design will be adjusted with issue #80

## Screenshots / GIFs (if applicable)

image

## Checklist

- [x] I have tested these changes locally
- [x] I have ensured my code follows the project's coding style
- [ ] I have updated the documentation accordingly
- [x] My changes do not introduce any new warnings or errors
- [x] All tests pass successfully
- [ ] I have added/updated unit tests (if necessary)

### CODE REVIEW

**Comment by kylescudder** [comment]
@JacobHeldt could these CSS changes please be done using tailwindcss as this is the CSS framework we are using with the project. 

Can you please also make sure that your changes follow the linting conversions (tabs etc) so it is easier to tell what has actually been changed, and what is just refactoring?

Cheers

File: components/Footer/Footer.tsx [hunk 1]
```diff
@@ -55,128 +60,88 @@ const Footer = ({
...
 	}
 
+
 	return (
-		<div className="footer">
-			{/* Footer logo */}
-			<div className="footer-logo">
-				{/* SVG Logo */}
-				<svg width="75" height="75" viewBox="0 0 446 445" fill="none"
+		<div className="bg-[#f2f7ff] grid grid-cols-3 lg:grid-cols-5 gap-4 px-6 lg:px-[150px] py-6 items-center">
```
Comment (kylescudder) [inline components/Footer/Footer.tsx:65]: Can we please use the tailwind configured `bg-zircon-50` for this?
image

**Review by kylescudder — CHANGES_REQUESTED** [review]
There seems to be a number of linting issues that need resolving, can you please fix these so we can tell what is a real change to a line, and what is a linting change (tabs, line length etc)

File: components/Footer/Footer.tsx [hunk 2]
```diff
@@ -55,70 +54,21 @@ const Footer = ({
 		scrollToRef(ref)
 	}
 
+
 	return (
-		<div className="footer">
+		<div className="bg-zircon-50 grid grid-cols-3 lg:grid-cols-5 gap-4 px-6 lg:px-[150px] py-6 items-center">
```
Comment (kylescudder) [inline components/Footer/Footer.tsx:59]: This line is 113 characters long, the linting limit is 110, please sort this? I know it sounds picky sorry.

File: components/Footer/Footer.tsx [hunk 3]
```diff
@@ -55,70 +54,21 @@ const Footer = ({
...
-          217.091V267.091C280.44 267.091 309.997 237.534 309.997 201.073H259.997ZM243.98 185.055C252.826
-          185.055 259.997 192.227 259.997 201.073H309.997C309.997 164.613 280.44 135.055 243.98
-          135.055V185.055ZM227.962 201.073C227.962 192.227 235.133 185.055 243.98 185.055V135.055C207.519
-          135.055 177.962 164.613 177.962 201.073H227.962ZM243.98 217.091C235.133 217.091 227.962 209.919
-          227.962 201.073H177.962C177.962 237.534 207.519 267.091 243.98 267.091V217.091Z" fill='#3A86FF'
-					fillOpacity="0.9" mask="url(#path-4-inside-1_387_2)" />
-				</svg>
-				<h3 className="footer-title">
-          StudyCrew
+				<img src="/assets/Logo.svg" alt="Logo" className='w-10'/>
```
Comment (kylescudder) [inline components/Footer/Footer.tsx:63]: Can you please use the `next/image` component for this? It should look like this:

```javascript
import Image from 'next/image'
import Logo from '../../public/assets/Logo.svg'

<Image alt='Logo' src={Logo} className='w-10' />
```

**Comment by kylescudder** [comment]
Has something happened to the mobile footer design?

image

**Review by kylescudder — CHANGES_REQUESTED** [review]

**Comment by JacobHeldt** [comment]
> Has something happened to the mobile footer design?
> 
> image

Issue #80 addresses the spacing issues of the footer on mobile. I could also resolve the mobile problems in this pull request, if that is preferred.

**Review by kylescudder — APPROVED** [review]

**Pull request merged**